### PR TITLE
imp notes

### PR DESCRIPTION
**Pull request by Ronald-codecreator**
imp notes

### CODE REVIEW

**Comment by Ronald-codecreator** [comment]
Imp notes for hibernate access diff schema

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,5 @@ build/
 
 ### VS Code ###
 .vscode/
+pom.xml
```
Comment (ricson-dev) [inline .gitignore:34]: why are these changes required

Comment (Ronald-codecreator) [reply .gitignore:34]: I have postgres so dont want to create collision

Comment (ricson-dev) [reply .gitignore:34]: OK

**Comment by ricson-dev** [comment]
OK
